src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form form-big"
+          data-qa="big"
+          action="/"
+    >
+      <img class="form__img-big"
+           src="images/Search.svg"
+           alt="search"
+      >
+
+      <input class="form__input form__input-big"
+             data-qa="keypress"
+             type="text"
+             placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form class="form form-small"
+          data-qa="small"
+          action="/"
+    >
+      <img class="form__img-small"
+           src="images/Search.svg"
+           alt="search"
+      >
+
+      <input class="form__input form__input-small"
+             type="text"
+             placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/77551820/127e48c5-efda-4b99-8ae2-abc18852640f">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**